lib/test.json

```diff
@@ -0,0 +1,42 @@
+{
+	"composite_facts":[
+		{
+			"id": "flu",
+			"operator": "AND",
+			"facts": ["headache", "high_temperature"]
+		},
+		{
+			"id": "indisgestion",
+			"operator": "AND",
+			"facts": ["bloating", "stomachache"]
+		}
+	],
+	"facts":[
+		{
```

Comment:
looks like `id` is redundant. i dont think it's necessary to create the fact id explicitly. we can do it implicitly